actors/miner/src/monies.rs

```diff
@@ -173,41 +185,37 @@ pub fn pledge_penalty_for_termination_lower_bound(
     )
 }
 
-/// Penalty to locked pledge collateral for the termination of a sector before scheduled expiry.
-/// SectorAge is the time between the sector's activation and termination.
-#[allow(clippy::too_many_arguments)]
+// TODO: write better description, mention FIP-0098
 pub fn pledge_penalty_for_termination(
-    day_reward: &TokenAmount,
+    initial_pledge: &TokenAmount,
     sector_age: ChainEpoch,
-    twenty_day_reward_at_activation: &TokenAmount,
-    network_qa_power_estimate: &FilterEstimate,
-    qa_sector_power: &StoragePower,
-    reward_estimate: &FilterEstimate,
-    replaced_day_reward: &TokenAmount,
-    replaced_sector_age: ChainEpoch,
+    fault_fee: &TokenAmount,
 ) -> TokenAmount {
-    // max(SP(t), BR(StartEpoch, 20d) + BR(StartEpoch, 1d) * terminationRewardFactor * min(SectorAgeInDays, 140))
-    // and sectorAgeInDays = sectorAge / EpochsInDay
-    let lifetime_cap = TERMINATION_LIFETIME_CAP * EPOCHS_IN_DAY;
-    let capped_sector_age = std::cmp::min(sector_age, lifetime_cap);
-
-    let mut expected_reward: TokenAmount = day_reward * capped_sector_age;
-
-    let relevant_replaced_age =
-        std::cmp::min(replaced_sector_age, lifetime_cap - capped_sector_age);
-
-    expected_reward += replaced_day_reward * relevant_replaced_age;
-
-    let penalized_reward = expected_reward * TERMINATION_REWARD_FACTOR_NUM;
-    let penalized_reward = penalized_reward.div_floor(TERMINATION_REWARD_FACTOR_DENOM);
+    // Use the _percentage of the initial pledge_ strategy to determine the termination fee.
+    let termination_fee =
+        (initial_pledge * TERM_PENALTY_PLEDGE_NUM).div_floor(TERM_PENALTY_PLEDGE_DENOM);
+
+    // There are a few special cases to consider where the termination fee must be tweaked to
+    // maintain the current network conditions.
+
+    // 1. We need to ensure that the final termination fee is always at least 105% of the fault fee for
+    //    the same sector.
+    let fault_fee_cap = (fault_fee * FAULT_FEE_MULTIPLE_NUM).div_floor(FAULT_FEE_MULTIPLE_DENOM);
+    let termination_fault_max = cmp::max(termination_fee, fault_fee_cap);
+
+    // 2. We need to ensure linear growth of the termination fee over time, up to a cap. The
+    //    details of this growth are defined in FIP-0098 design rationale.
+    let sector_age_in_days = sector_age / EPOCHS_IN_DAY;
+    let penalty_ramped = cmp::min(
+        (&termination_fault_max * sector_age_in_days).div_floor(TERMINATION_LIFETIME_CAP),
+        termination_fault_max,
+    );
 
+    // Ensure the termination fee is no less than the minimum termination fee pledge.
```

Comment:
This comment doesn't make sense to me

Reply:
These tests were superseded by the tests outlined by the FIP: https://github.com/filecoin-project/builtin-actors/blob/c1124049abdfe42418d23450782aa7d47410434a/actors/miner/tests/pledge_penalty_for_termination.rs